### PR TITLE
edit small bugs

### PR DESCRIPTION
**Pull request by FlyingDonkeys**

### CODE REVIEW

File: src/main/java/tahub/contacts/model/ModelManager.java [hunk 1]
```diff
@@ -170,8 +169,8 @@ public void addPerson(Person person) {
     @Override
     public void setPerson(Person target, Person editedPerson) {
         requireAllNonNull(target, editedPerson);
-
         addressBook.setPerson(target, editedPerson);
+        updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```
Comment (FlyingDonkeys) [inline src/main/java/tahub/contacts/model/ModelManager.java:173]: for consistency

File: src/main/java/tahub/contacts/model/ModelManager.java [hunk 2]
```diff
@@ -195,13 +194,13 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
...
     }
 
     @Override
-    public void setScaListFilePath(Path courseListFilePath) {
-        requireNonNull(courseListFilePath);
-        userPrefs.setCourseListFilePath(courseListFilePath);
+    public void setScaListFilePath(Path scaListFilePath) {
+        requireNonNull(scaListFilePath);
+        userPrefs.setScaListFilePath(scaListFilePath);
     }
```
Comment (FlyingDonkeys) [inline src/main/java/tahub/contacts/model/ModelManager.java:205]: @miloaisdino probably copied wrongly last time, doesnt seem to impact functionality though

File: src/main/java/tahub/contacts/model/studentcourseassociation/StudentCourseAssociationList.java [hunk 3]
```diff
@@ -223,9 +223,10 @@ public void set(List<StudentCourseAssociation> scaList) {
      */
     public void remove(StudentCourseAssociation toRemove) {
         requireNonNull(toRemove);
-        if (!internalList.remove(toRemove)) {
+        if (!internalList.contains(toRemove)) {
             throw new RuntimeException("SCA not found in list");
         }
+        internalList.remove(toRemove);
```
Comment (FlyingDonkeys) [inline src/main/java/tahub/contacts/model/studentcourseassociation/StudentCourseAssociationList.java:229]: did not seem to do any removal

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -965,3 +965,4 @@ Expected: Attendance cleared. Success message shown.
 
 ## **Appendix: Planned Enhancements**
 1. Add a `course-list` command to list all courses so that user can view courses in the system.
+2. Add support for more Course Codes and Tutorial IDs by changing the regular expressions (i.e:`VALIDATION_REGEX`) used to accommodate other valid course codes and tutorial ids.
```
Comment (FlyingDonkeys) [inline docs/DeveloperGuide.md:968]: Linked to the future enhancement stated in UG.

[the remark this answers is not in the document]
Comment (FlyingDonkeys) [reply docs/DeveloperGuide.md:966]: there are valid course codes such as ```LAJ1201``` which was pointed as a bug by some other people. our regex currently doesnt allow for that...

**Review by Zeepheru — APPROVED** [review]
LGTM

**Review by FlyingDonkeys — COMMENTED** [review]
made the changes

**Pull request merged**